### ISSUE TITLE
🐛 [BUG] - orientation of gravity sensor is reversed

### ISSUE THREAD

**Issue by Cathgao** [BlissRoms-x86/support#44]
### Description

Installed Bliss-Go 14.10.1 or 15.9 on my tablet PC(teclast X3 plus).
When boot into launcher, gravity sensor start to work and rotate the screen. In Landscape the direction is reversed but Portrait is correct
![QQ图片20240221112449](https://github.com/BlissRoms-x86/support/assets/21031564/472f6b0a-c354-46eb-b448-8f72ed5d0a73)
![QQ图片20240221112452](https://github.com/BlissRoms-x86/support/assets/21031564/095138c6-796c-4faf-a502-5c27b38312cd)
![QQ图片20240221112455](https://github.com/BlissRoms-x86/support/assets/21031564/fd593436-a843-4994-a0f9-479f62e4463a)

### Reproduction steps

```bash
1. Install Bliss
2. Boot in
3. Disable `Set Orientation` App
4. See error
```

### Screenshots

```bash
![DESCRIPTION](LINK.png)
```

### Category

Display

### OS Version

14.x, 15.x

**Comment by Cathgao** [comment]
:/data/data/com.termux/files/home # cat /sys/class/dmi/id/uevent
MODALIAS=dmi:bvnTECLAST:bvrG4K3_A1tPAD3.01:bd08/25/2017:br5.12:efr14.4:svnTECLAST:pnX3Plus:pvrDefaultstring:rvnTECLAST:rnX3Plus:rvrDefaultstring:cvnDefaultstring:ct30:cvrDefaultstring:skuG4K3_A1:

**Comment by Cathgao** [comment]
please tell me how can I test the `init.sh` without compile the Android system

**Comment by Cathgao** [comment]
![QQ图片20240222092635](https://github.com/BlissRoms-x86/support/assets/21031564/a9d3762b-3e04-46ea-9046-a33cdb3558db)

**Comment by Xtr126** [comment]
> please tell me how can I test the `init.sh` without compile the Android system

By remounting system as r/w and modifying the files. All that is o answered on the documentation. https://docs.blissos.org/knowledgebase/troubleshooting/remount-system-as-read-write/

**Comment by electrikjesus** [comment]
It's much easier to test settings with Bass builds: https://docs.blissos.org/configuration/configuration-through-command-line-parameters/#iio-options-configuration

**Comment by Cathgao** [comment]
I add  SET_IIO_ACCEL_X_OPT_SCALE=-1 in grub entry but nothing changed

> It's much easier to test iio settings with Bass builds: https://docs.blissos.org/configuration/configuration-through-command-line-parameters/#iio-options-configuration

**Comment by electrikjesus** [comment]
> I add  SET_IIO_ACCEL_X_OPT_SCALE=-1 in grub entry but nothing changed
> 
I was trying to say that only the Bass builds come with that option so far.

**Comment by Cathgao** [comment]
https://sourceforge.net/projects/blissos-dev/files/BlissBass/x86_64/Go/foss/20230814/
download from here?

> > I add  SET_IIO_ACCEL_X_OPT_SCALE=-1 in grub entry but nothing changed
> 
> I was trying to say that only the Bass builds come with that option so far.

**Comment by electrikjesus** [comment]
No, new builds were moved to their own space. https://bliss-bass.blisscolabs.dev/

**Comment by Cathgao** [comment]
Thanks. I have tried new Bass builds but it completely unusable on my tablet PC: booting over 3 minutes, touch screen unavailable and adding `SET_IIO_ACCEL_X_OPT_SCALE=-1` in grub menu still change nothing.
I wish this bug will be fixed in future BlissOS and I am very happy to use.
